specification/implementation/metadata/general-requirements/index.md

```diff
@@ -1,66 +1,65 @@
+The primary consideration when considering metadata requirements is, as with the rest of this specification, that of interoperability. That is to say that we focus on high-level technical interoperability and tasks which allow an Information Package to be prepared, transferred and received regardless of the institutions and tools involved. These tasks include:
 
 - Identifying uniquely an Information Package and its components;
 - Validating an Information Package;
 - Validating the contents of an Information Package;
-- Proving the authenticity of the Information Package;
+- Establishing the authenticity of the Information Package;
 - Accessing the contents of an Information Package.
 
-In more technical terms the CSIP makes an effort to control metadata which allows any tool or user to negotiate the data and metadata components of the package (i.e. packaging metadata), to validate that no component has come to harm during transfer or preservation (i.e. fixity information), to understand the processes behind the creation and management of the package (i.e. provenance and preservation metadata) and finally to understand how the data within the package could be accessed (i.e. representation information).
+In technical terms, the CSIP tries to control metadata needed by tools or users to:
 
-Most crucially, we regard descriptive metadata and most of detailed technical metadata to not belong in the scope of the CSIP. As such, the CSIP itself does not aim to provide detailed semantic interoperability between different systems. However, as noted in Section 1.2, implementers are welcome to use the
-construct of Content Information Type Specifications to achieve an even higher level of interoperability.
+- navigate data and metadata components within the package (i.e. packaging metadata);
+- to validate that no component has been damaged during transfer or preservation (i.e. fixity information);
+- to understand the processes used when creating and managing the package (i.e. provenance and preservation metadata); and
+- to understand how the data within the package can be accessed (i.e. representation information).
 
-We implement the core metadata requirements with METS (Metadata Encoding & Transmission Standard, <http://www.loc.gov/standards/mets/>). In this specification we describe the core elements used, more elements are available in the METS standard and can be used in the own implementation.
+We regard descriptive metadata and most detailed technical metadata as outside the scope of the CSIP. As such, the CSIP does not aim to provide detailed semantic interoperability between different systems. However, as noted in Section 1.2, implementers are welcome to use the Content Information Type Specifications to achieve a  higher level of interoperability.
```

Comment:
```suggestion
We regard descriptive metadata and most of the detailed technical metadata as outside the scope of the CSIP. As such, the CSIP does not aim to provide detailed semantic interoperability between different systems. However, as noted in Section 1.2, implementers are welcome to use the Content Information Type Specifications to achieve interoperability on a more detailed level.
```

Reply:
We use the vertical metaphor so that "high" means "general" and "low" means "detailed," e.g "high-level technical interoperability" in the 1st paragraph of § 5.1, Figure 3 in § 1.2, Figure 4 in § 1.3. So here it should be "lower level" or "deeper" interoperability, or simply "on a more detailed level."